zypp/target/RpmPostTransCollector.cc

```diff
@@ -104,14 +132,14 @@ namespace zypp
 
 	    if ( ret != 0 || ! scriptmsg.empty() )
 	    {
-	      const std::string & pkgident( script.substr( 0, script.size()-6 ) );	// strip tmp file suffix
-
 	      if ( ! scriptmsg.empty() )
 	      {
 		str::Str msg;
 		msg << "Output of " << pkgident << " %posttrans script:\n" << scriptmsg;
 		historylog.comment( msg, true /*timestamp*/);
-		JobReport::info( msg );
+
+		//only show the actual message, package name is already in the progress line before
+		JobReport::info( scriptmsg );
```

Comment:
Please stay with the complete `msg`. 

Sooner or later the progress line will go away in zypper, and I'd like to get the message colored in zypper (the same way as the 'additional rpm output'). At least in theory YAST and others could respond to the JobReport as well, so the block should be complete.

Reply:
Done